nntrainer/layers/input_layer.h

```diff
@@ -82,7 +82,7 @@ class InputLayer : public Layer {
   /**
    * @copydoc Layer::supportInPlace()
    */
-  bool supportInPlace() const override { return true; }
+  bool supportInPlace() const override { return false; }
```

Comment:
Any reason to set this false as a default?

Reply:
It's for checking dut to input layer's inplace check had a bug.
please chekc modified input layer check code.